concent_api/core/views.py

```diff
@@ -617,6 +619,16 @@ def deserialize_message(raw_message_data):
         raise Http400("Unable to deserialize Golem Message: {}.".format(exception))
 
 
+def validate_report_computed_task_time_window(report_computed_task):
```

Comment:
Is there a unittest for it?

Reply:
Yeah. Would be a good idea to start adding unit tests right now. This pull request is so it's a good place to start.

Reply:
Yes there is. `test_send_should_return_http_400_if_task_to_compute_younger_than_report_computed` in `test_core_views.py`